### PR TITLE
Enable Already Completed button only when appropriate - PMT #100689

### PR DESCRIPTION
**Pull request by nikolas**
This adds a new method `highest_module_accessed` to the Participants,
and adds some JavaScript that enables the "Already Completed"
radio button on the sign-in form only when the dropdown is
populated.

### CODE REVIEW

File: media/js/src/views/sign-in-participant.js [hunk 1]
```diff
@@ -14,48 +15,97 @@ define([
...
+         * @param {jQuery element} $el - The <select> element.
+         */
+        _enableAlreadyCompletedDropdown: function($el) {
+            $el.prop('disabled', false);
+        },
+
+        _refreshDisabledState: function($el) {
             var $parent = $el.closest('.worth-participant-destination');
             if ($parent.length > 0) {
+                var $select;
```
Comment (sdreher) [inline media/js/src/views/sign-in-participant.js]: nit - the elt could be initialized here.

Comment (nikolas) [inline media/js/src/views/sign-in-participant.js]: Thanks, I'll change that.

File: worth2/main/generic/models.py [hunk 2]
```diff
@@ -78,7 +78,10 @@ def percent_complete_hierarchy(self):
             return 0
 
     def time_spent(self):
```
Comment (sdreher) [inline worth2/main/generic/models.py]: Is the time_spent function in use?

Comment (nikolas) [inline worth2/main/generic/models.py]: No, this model was copied over from MeaningfulConsent -- https://github.com/ccnmtl/meaningfulconsent/blob/master/meaningfulconsent/main/models.py#L99

Maybe we can remove this function? I figured that eventually we might want to move this model into django-pagetree.

Comment (sdreher) [inline worth2/main/generic/models.py]: Yeah, probably a good idea to remove. I have a more refined calculation of time spent here: https://github.com/ccnmtl/worth2/blob/master/worth2/main/reports.py#L91. It's more applicable to Worth2 as the intervention spans a much greater period of time. Meaningful Consent is meant to be taken all at once, so the calc was less complicated.

Comment (nikolas) [inline worth2/main/generic/models.py]: I see. I added a commit here removing it.

**Pull request merged**